test/jit/test_class_type.py

```diff
@@ -1013,6 +1013,79 @@ def foo():
             y.my_list = new_list
             return y
 
+    def test_default_args(self):
```

Comment:
This test case doesn't seem to exercise the logic of padding default values (in case of a fn only has part of its arguments having default values). Could you add one?

Reply:
Do you mean something like:
```
        class ClassWithDefaultArgs:
            def __init__(
                self,
                a: int,
                b: List[int] = [1, 2, 3],
            ):
```

Reply:
Yep. 

On a related note, maybe we can have a test covering the exception thrown with mutable default values as well?